lib/jwt/base64.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'base64'
+
+module JWT
+  # Base64 helpers
+  class Base64
+    class << self
+      def url_encode(str)
```

Comment:
Do you really want to add two more methods to the public API of this gem that handle Base64 encoding? This kind of thing should probably be kept private.